yarn-project/end-to-end/src/e2e_l1_to_l2_msg.test.ts

```diff
@@ -124,11 +123,11 @@ describe.skip('e2e_l1_to_l2_msg', () => {
     return contract;
   };
 
-  it('Should be able to consume an L1 Message and mint a non native token on L2', async () => {
-    const initialBalance = 1n;
-    const ownerAddress = accounts[0];
-    const owner = await aztecRpcServer.getAccountPublicKey(ownerAddress);
-    const deployedContract = await deployContract(initialBalance, pointToPublicKey(owner));
+  it('Milestone 2.2: L1->L2 Calls', async () => {
+    const initialBalance = 10n;
+    const [ownerAddress, receiver] = accounts;
+    const ownerPub = await aztecRpcServer.getAccountPublicKey(ownerAddress);
+    const deployedContract = await deployContract(initialBalance, pointToPublicKey(ownerPub));
```

Comment:
can you rename this to `deployedL2Contract`? Since we have a contract in L1 and another on L2, can get quite confusing